ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -376,6 +366,27 @@ export default React.memo<Props>(function VideoJs(props: Props) {
         vjsPlayer = window.player;
       }
 
+      // hide unused elements on livestream
```

Comment:
http://localhost:9090/@realityunknown:2/replay:d0

<img width="602" alt="image" src="https://user-images.githubusercontent.com/64950861/176392433-a9eeee5a-2cda-486f-8341-6f492274e335.png">

Also happening in production.  Seems specific to this stream.

Reply:
[Potential explanation] Seems like the streamer already stopped sending, but server still considers them as streaming.